WBooks/DetailBook/DetailBookViewController.swift

```diff
@@ -0,0 +1,100 @@
+//
+//  DetailBookViewController.swift
+//  WBooks
+//
+//  Created by daniel.f.munoz on 25/08/21.
+//
+
+import UIKit
+
+class DetailBookViewController: UIViewController {
+    
+    private lazy var detailBookView = DetailBookView()
+    private let detailBookViewModel: DetailBookViewModel
+
+    init(detailBookViewModel: DetailBookViewModel) {
+        self.detailBookViewModel = detailBookViewModel
+        super.init(nibName: nil, bundle: nil)
+    }
+
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        self.title = NSLocalizedString("TITLE_DETAILBOOK", comment: "")
+        detailBookView.tableView.delegate = self
+        detailBookView.tableView.dataSource = self
+        detailBookView.bookImage.imageFromUrl(urlString: detailBookViewModel.book.image)
+        detailBookView.titleLabel.text = detailBookViewModel.book.title
```

Comment:
We can do a instance of detailBookViewModel.book to write less 😉

Reply:
Done